config/cobbler/distro_signatures.json

```diff
@@ -1024,6 +1007,23 @@
     "boot_files":[],
     "boot_loaders":{"ppc64le":["grub2"]}
    },
+   "sles12sp4": {
+    "signatures":["suse"],
+    "version_file":"(sles|SLES-for-VMware|SLES_SAP)-release-12.4-(.*).rpm",
+    "version_file_regex":null,
+    "kernel_arch":"kernel-(.*)\\.rpm",
+    "kernel_arch_regex":null,
+    "supported_arches":["i386","i586","x86_64","ppc64le"],
+    "supported_repo_breeds":["yum"],
+    "kernel_file":"linux[64.gz]?",
+    "initrd_file":"initrd[64]?",
+    "isolinux_ok":false,
+    "default_autoinstall":"sample_autoyast.xml",
+    "kernel_options":"dhcptimeout=120, net.ifnames=1, install=$tree",
```

Comment:
The whole default_autoinstall vs default_kickstart
and relative (sample.seed) vs absolut path (/var/lib/cobbler/kickstarts/sample_end.ks)
is  a huge mess in config/cobbler/distro_signatures.json.
This certainly should be fixed in a separate submit request.
I guess everything should be relative and default_autoinstall.
And the directory nowadays is: /var/lib/cobbler/autoinstall_templates, but in my installation only /var/lib/cobbler/templates exists...

Ideally this is fixed up by a separate commit.